README.md

```
@@ -4,7 +4,7 @@ This is an app for [Hack Club](https://hackclub.com)
 
 It is currently under development, but you can request features in the [Hack Club Slack](https://hackclub.com/slack/), channel [**#hackclub-mobile**](https://app.slack.com/client/T0266FRGM/C09NTG60T8X)
 
-Docs availible at: https://hcmobile.dino.icu
+Docs availible at: https://hcmobile.veridionstudios.org
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)